indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -463,9 +432,24 @@ def _process(self):
         return jsonify_data(**self.list_generator.render_list())
 
 
-class RHRegistrationsExportAttachments(RHRegistrationsExportBase):
+class RHRegistrationsExportAttachments(RHRegistrationsExportBase, ZipGeneratorMixin):
     """Export registration attachments in a zip file"""
 
+    def _prepare_folder_structure(self, attachment):
+        registration = attachment.registration
+        regform_title = secure_filename(attachment.registration.registration_form.title, 'registration_form')
+        registrant_name = secure_filename("{}_{}".format(registration.get_full_name(),
+                                          unicode(registration.friendly_id)), registration.friendly_id)
+        file_name = secure_filename("{}_{}_{}".format(attachment.field_data.field.title, attachment.field_data.field_id,
+                                                      attachment.filename), attachment.filename)
+        path = os.path.join(*self._adjust_path_length([regform_title, registrant_name, file_name]))
+        return path
```

Comment:
i'd just return it in the previous line. no need in assigning it just to return it and not do anything else with it